src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -591,4 +615,31 @@ protected function note($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * Set the event dispatcher instance.
+     *
+     * @param  \Illuminate\Contracts\Events\Dispatcher  $dispatcher
+     * @return void
+     */
+    public static function setEventDispatcher(Dispatcher $dispatcher)
```

Comment:
... but you've declared it as a static function.

Reply:
Good spot. My mistake. I'll fix that. Interesting that it raised no issues for me locally. Guess PHP is forgiving.

Reply:
Fixed.